spec/features/form_sections/teacher_training_data/placements/editing_trainee_placements_spec.rb

```diff
@@ -115,6 +137,7 @@ def and_i_am_on_the_add_placement_page
   end
 
   def when_i_select_an_existing_school(school: school_one)
+    find(:xpath, "//input[@name='placement[school_search]']").set("")
```

Comment:
In real-life this field name gets mangled if JS is enabled so it doesn't end up being posted in the form data to the server.